frontend/tsconfig.dev.json

```diff
@@ -0,0 +1,9 @@
+{
+    "extends": "./tsconfig.json",
+    "compilerOptions": {
+      "paths": {
+        "@streamlit/lib": ["lib/src"],
```

Comment:
Why do we still need this?

Reply:
This is the original pathing alias that tells typescript how to resolve our paths.